<?xml version="1.0"?>
<launch>

    <!-- Eye_decoder arguments -->
        <arg name="cam_source"                  default='0'/>
        <arg name="blink_threshold"             default='0.52'/>
        <arg name="rate"                        default='256'/>
        <arg name="show_frame"                  default='True'/>

    <!-- eye_decoder node -->
    <node name="eye_decoder" pkg="eye_decoder" type="eye_decoder_node.py" output="screen">
        <param name="cam_source"          type="int"               value="$(arg cam_source)"/>
        <param name="rate"                type="int"               value="$(arg rate)"/>
        <param name="show_frame"          type="bool"              value="$(arg show_frame)"/>
        <param name="blink_threshold"     type="double"            value="$(arg blink_threshold)"/>
    </node>

</launch>
